Formula/kubesess.rb

```diff
@@ -0,0 +1,18 @@
+class Kubesess < Formula
+  desc "Manage multiple kubernetes cluster at the same time"
+  homepage "https://rentarami.se/posts/2022-08-05-kube-context-2/"
+  url "https://github.com/Ramilito/kubesess/archive/refs/tags/1.2.5.tar.gz"
+  sha256 "1ce69422fc29bde69ac02a209eb7b27a447598c39a3870759aa4c83e553b2e52"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "kubernetes-cli" => :test
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "USAGE:", shell_output("#{bin}/kubesess -h 2>&1")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Yeah, I felt the same but couldn't figure out a better test since this tool needs kubernetes-cli setup on the host to do anything. Took a look at a similar tool (kubectx) where they also do the -help test and thought they got an exempt?

Reply:
> Took a look at a similar tool (kubectx) where they also do the -help test and thought they got an exempt?

Maybe, but that doesn't mean that we're keen on adding another bad test.

Can we try to run something that we know will error and check that it fails as we expect it to?

Reply:
Understandable, found a way to make it work with a real case!

Reply:
Was I not supposed press the resolve conversation button?